src/shared/components/post/post-listing.tsx

```diff
@@ -282,16 +282,29 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
       return (
         <div className="ratio ratio-16x9 mt-3">
           <video
-            onLoadStart={linkEvent(this, this.handleVideoLoadStart)}
-            onPlay={linkEvent(this, this.handleVideoLoadStart)}
-            onVolumeChange={linkEvent(this, this.handleVideoVolumeChange)}
+            onLoadStart={linkEvent(this, this.handleMediaLoadStart)}
+            onPlay={linkEvent(this, this.handleMediaLoadStart)}
+            onVolumeChange={linkEvent(this, this.handleMediaVolumeChange)}
             controls
             aria-label={post.alt_text}
           >
-            <source src={post.embed_video_url ?? url} type="video/mp4" />
+            <source src={post.embed_video_url ?? url} />
```

Comment:
Specifying type here seems wrong as it may be webm or another format. Better let the browser figure it out.